### ISSUE TITLE
Select columns directly on the canvas.

### ISSUE THREAD

**Issue by F-Burning** [opentoonz/opentoonz#4641]
### Is your feature request related to a problem? Please describe

Currently you can't select columns directly from the canvas, which makes editing much faster.

### Describe the solution you'd like

Select columns when make clic with the mouse on the screen.

**Comment by flurick** [comment]
Interesting, the style picker tool can change layer. But only for toonz-raster and vector layers.

**Comment by artisteacher** [comment]
The code's already in the Animate tool's "All" mode. It can have the Pick value set to "Column". Then clicking on a drawing makes it active, as you'd like. I'm not sure why that's only available in the one mode. Perhaps due to clicking and dragging to rotate/move/skew instead of using a widget?

**Comment by F-Burning** [comment]
Curious, I didn't notice, thank you for notifying 👍 

Something that I am noticing is that this option does not select the column exactly, but the frame. For example, you can't select a column from the screen to copy and paste it if you haven't previously selected the column from the timeline.

Something that would also be nice would be the possibility to select several columns using the control key, or with some rectangular selection (this would be less precise I suppose)

**Comment by flurick** [comment]
Yes, could be used to inserting effect nodes from the canvas directly (via the search window)
